frontend/components/receipts/orders/submission/submission.spec.tsx

```diff
@@ -0,0 +1,50 @@
+import { render, screen } from '@testing-library/react'
+import { Submission } from '.'
+import { locators } from '../../../data-table/data-table'
+
+describe('SubmissionReceipt', () => {
+  it('renders nothing if the transaction is a pegged order', () => {
+    const { container } = render(
+      <Submission
+        transaction={{
+          orderSubmission: {
+            peggedOrder: {}
+          }
+        }}
+      />
+    )
+    expect(container).toBeEmptyDOMElement()
+  })
+  it('renders nothing if the transaction is an iceberg order', () => {
+    const { container } = render(
+      <Submission
+        transaction={{
+          orderSubmission: {
+            icebergOpts: {}
+          }
+        }}
+      />
+    )
+    expect(container).toBeEmptyDOMElement()
+  })
+  it('renders order table and badges', () => {
```

Comment:
looks to me like we are missing quite a few assertions here to cover the below AC